code/modules/admin/topic.dm

```diff
@@ -1413,11 +1479,11 @@
 			return
 
 		var/mob/M = locateUID(href_list["aroomwarp"])
-		if(!ismob(M))
-			to_chat(usr, "This can only be used on instances of type /mob")
+		if(!istype(M, /mob))
```

Comment:
Any reason you overrode all the instances of `ismob`?

Reply:
I think this was done when I was newer to DM and thought `ismob` required a non-null argument, but that's obviously not the case after testing.